xinference/model/llm/core.py

```diff
@@ -57,6 +61,81 @@ def __setitem__(self, key: str, value: Any):  # type: ignore
         super().__setitem__(key, value)
 
 
+def _to_prompt(
```

Comment:
Better write a mixin class instead of putting the util functions in the global scope. For the definition of mixin, see https://zhuanlan.zhihu.com/p/95857866

Reply:
done